tests/filecheck/transforms/convert-stencil-to-csl-stencil.mlir

```diff
@@ -3,217 +3,6 @@
 builtin.module {
 // CHECK-NEXT: builtin.module {
 
-  func.func @gauss_seidel(%a : !stencil.field<[-1,1023]x[-1,511]xtensor<512xf32>>, %b : !stencil.field<[-1,1023]x[-1,511]xtensor<512xf32>>) {
```

Comment:
@dk949 @emmau678 we removed a large chunk of the filecheck for testing purposes, but then accidentally committed and pushed. Will raise a PR to put this back in.

Reply:
cc @superlopuh